packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -210,6 +211,21 @@ export class TxnBuilderLoans extends P2WPKHTxnBuilder {
     )
   }
 
+  /**
+   * PaybackLoanV2 transaction.
+   *
+   * @param {PaybackLoanV2} paybackLoanV2 txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+
```

Comment:
```suggestion
```

Reply:
loans.paybackLoanV2 is tested in packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_payback_loan.test.ts
I have not declared instance type as `PaybackLoanV2`, That should be the reason for codcov failure, let me see it.

Reply:
@chanakasameera `packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/PaybackLoan.test.ts` equivalent for `PaybackLoanV2` needs to be added.

Reply:
Yup, @chanakasameera it's for sanity and regression proofing, FYI I created https://github.com/DeFiCh/jellyfish/issues/1295 to track that. Going to merge this in first.